internal/controllers/reconciler.go

```
@@ -0,0 +1,449 @@
+		if errors.Is(err, context.Canceled) {
+			c.logger.InfoContext(ctx, "Watch finished")
+			return
+		}
+		if err == nil {
+			c.logger.ErrorContext(
+				ctx,
+				"Watch failed",
+				slog.Any("error", err),
+			)
+		}
+		c.sleepAfterWatch(ctx)
```

golangci-lint (1.64.8)

Error return value of `c.sleepAfterWatch` is not checked

(errcheck)